### PR TITLE
Add documentation for Ruby SBOM.

### LINKED ISSUE #487: Add Ruby SBOM documentation
With the upcoming Ruby SBOM work nearing completion, we should add documentation for it here.

It will likely look similar to the Python and Golang documentation, though it will probably be simpler as there are no known caveats.

### PR DESCRIPTION
**Pull request by robdimsdale**
## Summary

Add documentation for the new SBOM feature in the Ruby buildpack.

Closes #487

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by robdimsdale** [comment]
@paketo-buildpacks/content-maintainers I don't know why the spellchecker is now complaining about `bundler` in a link, when it wasn't previously a problem. Can you help me resolve this?

**Comment by fg-j** [comment]
In this case, the spellchecker was configured to accept `Bundler` as a proper noun, but not `bundler`. I tweaked the regex in the dictionary to allow for both.

File: content/docs/howto/ruby.md [hunk 1]
```diff
@@ -212,12 +212,21 @@ To use this feature:
 1. Include an `app/assets` directory in your app source code
 1. Add the `rails` gem to your `Gemfile`
 
+## Access the software bill of materials
+The Ruby buildpack includes support for the software bill of materials (SBOM).
+Check out the [SBOM how-to documentation][how-to/SBOM] for details on how to
+access the SBOM supplied by the buildpacks.
+
+SBOMs will be generated for applications which leverage `Bundler`.
```
Comment (fg-j) [inline content/docs/howto/ruby.md:220]:
```suggestion
SBOMs will be generated for applications which leverage Bundler.
```

File: content/docs/reference/ruby-reference.md [hunk 2]
```diff
@@ -76,3 +76,27 @@ variable and its impact on your app.
...
+
+##  Software Bill of Materials
+The Ruby buildpack supports the full [software bill of
+materials][concepts/SBOM] (SBOM) in [Syft][format/syft],
+[CycloneDX][format/cyclonedx], and [SPDX][format/spdx] formats. The Ruby
+buildpack also includes limited support for the
+[Paketo-specific][format/paketo] SBOM format. This Paketo-specific SBOM format
+does not include information about the application dependencies.
+
+SBOMs will be generated for applications which leverage `Bundler`.
```
Comment (ForestEckhardt) [inline content/docs/reference/ruby-reference.md:88]:
```suggestion
SBOMs will be generated for applications which leverage Bundler.
```

Seeing the previous commit it seems this should also change.

**Review by TisVictress — APPROVED** [review]

**Review by sophiewigmore — APPROVED** [review]

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**